scylladb/sphinx-scylladb-theme
# Highlight home page on breadcrumbs

Closes #211 

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/ with your favorite browser, and navigate to an internal page. You will see the new breadcrumbs design:

![image](https://user-images.githubusercontent.com/9107969/135255474-b2175e29-319d-4e6a-864a-fa0107abd72c.png)
